### PR TITLE
Multiprocessing optimization fix

### PR DESCRIPTION
**Pull request by generall**
Create multiprocessing optimizer test + fix found issues

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally using ``cargo fmt`` command prior to submission?
3. [x] Have you checked your code using ```cargo clippy``` command?

### CODE REVIEW

File: lib/collection/src/collection_manager/optimizers/indexing_optimizer.rs [hunk 1]
```diff
@@ -184,14 +198,18 @@ mod tests {
 
         let locked_holder = Arc::new(RwLock::new(holder));
 
+        let excluded_ids = Default::default();
```
Comment (agourlay) [inline lib/collection/src/collection_manager/optimizers/indexing_optimizer.rs:201]: do we need a test with a non-empty `excluded_ids`?

**Review by agourlay — APPROVED** [review]
